pytesmo/metrics.py

```diff
@@ -59,16 +103,19 @@ def rmsd(o, p):
         Observations.
     p : numpy.ndarray
         Predictions.
+    ddof : int, optional
+        Delta degree of freedom.The divisor used in calculations is N - ddof,
+        where N represents the number of elements. By default ddof is zero.
 
     Returns
     -------
     rmsd : float
         Root-mean-square deviation.
     """
-    return np.sqrt(RSS(o, p) / len(o))
+    return np.sqrt(RSS(o, p) / (len(o) - ddof))
```

Comment:
This is actually an old bug because len(o) is a integer so division will result in an integer. Could you please add
```python
from __future__ import division
```
This should make it behave consistently in python2 and python3